Sources/ValCommand/ValCommand.swift

```diff
@@ -212,21 +247,49 @@ public struct ValCommand: ParsableCommand {
     try write(
       cxxModules.source, to: buildDirectory.appendingPathComponent(productName),
       loggingTo: &errorLog)
+      
+    var compiler = ""
+    switch customizeCXXCompiler {
+      case .clang: compiler = try find("clang++")
+      case .gcc: compiler = try find("g++")
+      case .msvc: compiler = try find("cl")
+    }
+    
+    let binaryPath = executablePath(outputURL, productName)
 
-    let clang = try find("clang++")
+    #if os(Windows)
+      if customizeCXXCompiler == .msvc {
+        var arguments =  [ buildDirectory.appendingPathComponent(productName + ".cpp").path,
+          "/link", "/out:" + binaryPath]
+        
+        if ccFlags != ["Empty"] {
```

Comment:
```suggestion
        if !ccFlags.isEmpty {
```
Assuming we can declare `ccFlags: [String] = []` as I suggested.